autorel/changelog_generator/parser_config.py

```diff
@@ -0,0 +1,55 @@
+## Register your parsing functions here
+
+import re
+
+
+def pull_id_parser(msg_body):
+    """
+        Parses pull request id from commit message
+        => msg : Merge pull request #1234 from username/branch
+        => pull_id : 1234
+    """
+    if msg_body.startswith('Merge pull'):
+        tokens = msg_body.split()
+        pull_id = tokens[3][1:]
+        return pull_id, "pull_id"
+
+
+def pull_branch_parser(msg_body):
+    """
+        Parses branch name from commit message
+        => msg : Merge from user/master
+        => branch_name : user/master
+    """
+    if msg_body.startswith('Merge branch'):
+        tokens = msg_body.split()
+        pull_branch = tokens[2]
+        return pull_branch, "merged_branch"
```

Comment:
It's not universal, for example: https://github.com/balabit/syslog-ng/commit/4067982f8edc413dc311c08c6b116df1ee334631